src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/illegaltoken/InputIllegalTokensCheckBlockCommentBegin.java

```diff
@@ -7,7 +7,7 @@
 
 package com.puppycrawl.tools.checkstyle.checks.coding.illegaltoken;
 
-/** // violation
+/** // violation 'IllegalToken'
```

Comment:
https://checkstyle.org/checks/coding/illegaltoken.html#Violation_Messages


`'is not allowed.'`